packages/runner-ct/cypress/component/utils.ts

```diff
@@ -0,0 +1,18 @@
+import State from '../../src/lib/state'
+
+export const fakeConfig = { projectName: 'Project', env: {}, isTextTerminal: false } as any as Cypress.RuntimeConfigOptions
```

Comment:
I realize this is copied, but you should always cast to `unknown`, not `any`. `any` should rarely, if ever, appear in our codebase.

Reply:
Done